hrjobcontract/tests/phpunit/CRM/Hrjobcontract/Import/Parser/ApiTest.php

```diff
@@ -47,6 +53,22 @@ function setUp() {
     ];
   }
 
+  function createPayScale() {
```

Comment:
That would be nice, I've heard different things on this since in another review I was asked to add a teardown method. Do you know if it's reset at the end of each test method, or is it by class?